src/Discord.Net.WebSocket/Entities/Users/SocketThreadUser.cs

```diff
@@ -53,6 +53,20 @@ public override string AvatarId
             internal set => GuildUser.AvatarId = value;
         }
 
+        /// <inheritdoc/>
+        public override string BannerId
+        {
+            get => GuildUser.BannerId;
+            internal set => GuildUser.BannerId = value;
```

Comment:
why add setters if they are not used?

Reply:
idk, avatar property too has setter

Reply:
because it implementation of interfaces/classes

Reply:
Any property like that will be updated in the class that owns it, therfor we dont need to modify any properties out of sync with that guild user. Could you please remove it

Reply:
ok

Reply:
but how 🤔

Reply:
This class created by you, and inherits `SocketUser`. `AvatarID` and other props also has setters, because its implementation of `SocketUser`.

Reply:
just dont override it...

Reply:
nvm its good ig.